neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
Well trunk is a commonly used term in neuroscience (at least to my knowledge, so please correct me if this is not accurate). It is True that the way this term is used is generally very confusing...

So, in general...
Trunk is the initial section (so no segment and no point). When we refer to the orientation of a tree (elevation and azimuth) we generally refer to the vector (initial point - soma center) so the initial orientation. So when you refer to trunk elevation and trunk azimuth to me this is equivalent to the tree elevation and azimuth which implies a reference to the first point.

So, we can use any of the following, as long as the definitions are clear!
trunk_ (elevation, azimuth, direction)
tree_ (elevation, azimuth, direction)
initial_ (elevation, azimuth, direction)